docker-compose.yml

```diff
@@ -441,7 +442,7 @@ services:
       - GENNY_KOGITO_SERVICE_URL=http://alyson2.genny.life:${GADAQ_PORT}
       - GENNY_KOGITO_DATAINDEX_HTTP_URL=http://alyson.genny.life:${DATAINDEX_PORT}
       - GENNY_GADAQ_SERVICE_URL=http://alyson2.genny.life:${GADAQ_PORT}
-      - GENNY_KEYCLOAK_URL=https://keycloak.gada.io/auth
+      - GENNY_KEYCLOAK_URL=https://keycloak-bali.gada.io
```

Comment:
as you have env_file settings above , you can read GENNY_KEYCLOAK_URL from ${ENV_FILE} by saving GENNY_KEYCLOAK_URL to genny.env

Reply:
Fixed